TriblerGUI/widgets/trustpage/js/request_data.js

```diff
@@ -20,7 +25,7 @@ function get_node_info(public_key) {
     }
 
     response.onload = function() {
-        console.log(response.responseText);
+        callback(response.responseText);
```

Comment:
Instead of creating a function which calls the passed callback, can't you assign the callback directly to `response.onload`, while passing it's argument(s)?